### PR TITLE
Updated performance tests to use DbClient 

### PR DESCRIPTION
**Pull request by Nana-EC**
**Detailed description**:
Recently I realized the performance tests weren't working because the db inserts and deletes weren't working.
It seems with our r2dbc update the enforcement for transactional statements was enforced.

As such I took the opportunity to 
- update the ConnectionHandler to use DatabaseClient.
- Added the logic to ensure a topic entity exists before adding to it. This is since the gRPC layer added a check for it
- Applied PR feedback to update the acceptance test hcs feature scenarios

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java [hunk 1]
```diff
@@ -66,9 +64,54 @@ private PostgresqlConnectionFactory getConnectionFactory() {
...
-        return getConnectionFactory().create().block(Duration.ofMillis(500));
+    private DatabaseClient getClient() {
+        return DatabaseClient.create(getConnectionFactory());
+    }
+
+    public long createTopic() {
+        long topicNum = getNextAvailableTopicID();
+
+        String entityInsertSql = "insert into t_entities"
+                + " (id, entity_num, entity_realm, entity_shard, fk_entity_type_id, key, ed25519_public_key_hex)"
```
Comment (steven-sheehy) [inline hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java]: You don't need to insert all fields, just the required ones (entity_num, entity_realm, entity_shared, fk_entity_type_id). Id is auto generated. This keeps it from being too coupled.

File: hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java [hunk 2]
```diff
@@ -77,6 +120,10 @@ public void insertTopicMessage(int newTopicsMessageCount, long topicNum, Instant
             return;
         }
 
+        if (!topicExists(topicNum)) {
+            topicNum = createTopic();
```
Comment (steven-sheehy) [inline hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java]: These two can be combined with an `INSERT ... ON CONFLICT DO NOTHING`

File: hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java (hunk 1 above)
Comment (Nana-EC) [reply hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java]: Good point. I left out a few but I can drop more.

File: hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java (hunk 2 above)
Comment (Nana-EC) [reply hedera-mirror-grpc/src/test/java/com/hedera/mirror/grpc/jmeter/ConnectionHandler.java]: Will do. Will also add a createNewTopic to utilize other logic

**Pull request merged**